sdk/identity/identity/src/credentials/visualStudioCodeCredential.ts

```diff
@@ -176,7 +176,7 @@ export class VisualStudioCodeCredential implements TokenCredential {
 
     // We want to make sure we use the one assigned by the user on the VSCode settings.
     // Or just `Azure` by default.
```

Comment:
Should probably update the comment to refer to the same default name

Reply:
Oh! Thank you for spotting this!